### PR TITLE
fixed 6529: suggested num questions empty when no exercises selected

### PR DESCRIPTION
**Pull request by Ali-Kazmi**
### Summary
 On the CreateExamPage, added a new string (numQuestionsExceedNoExercises) which allows me to fix the bug where it was just showing blank instead of 0, as well as to match the exact string the issue requested. Used this string in the case where 0 questions are available. Appears to work in screenshots below (before, then after) 
Screen Shot 2020-12-16 at 12 27 33 PM
After: 
Screen Shot 2020-12-16 at 1 56 43 PM

…

### Reviewer guidance
 Please test this by checking the scenario where nothing is selected (which I have tested) and when something is selected (I had trouble testing this because I didn't know how to add content, so its worth testing). 

…

### References
https://github.com/learningequality/kolibri/issues/6529 

…

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/index.vue [hunk 1]
```diff
@@ -324,6 +324,12 @@
         if (!Number.isInteger(this.numQuestions)) {
           return this.$tr('numQuestionsBetween');
         }
+        if (this.availableQuestions == 0 || this.availableQuestions == null) {
+          return this.$tr('numQuestionsExceedNoExercises', {
```
Comment (jonboiser) [inline kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/index.vue:328]: You don't need to add this new `numQuestionsExceedNoExercises` message. You can reuse the existing `numQuestionsExceed` message like so:

```
this.$tr('numQuestionsExceed', {
            inputNumQuestions: this.numQuestions,
            maxQuestionsFromSelection: 0,
});
```

Comment (Ali-Kazmi) [reply kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/index.vue:328]: Interestingly when I tried that I was still seeing the issue. Also the the end of the string didn't match with what the issue asked for (they didn't want to say lower the number to 0). If we still want to change it though I can try again, just let me know!

**Comment by jonboiser** [comment]
@Ali-Kazmi Would you mind rebasing this PR on top of the `develop` branch? Because of how we translate UI messages, we cannot add a new message like `numQuestionsExceed` to the `release-v0.14.x` branch (that branch's translations are frozen).

**Comment by Ali-Kazmi** [comment]
Sure, i'll rebase it in a bit (sorry, still at work).

**Comment by Ali-Kazmi** [comment]
Ok, I just tried to rebase to develop and pushed it here (note I had to force push, but in the kolibri dev docs it says that is ok if its on my branch?). Someone should look over this carefully, as I haven't used rebase before. I also used git rebase --skip on a few conflicts (that seemed unrelated to my change, but i'm not sure). Should I change the target branch to develop? This is kind of a mess so it may just be easier to make a new pr straight from the develop branch with these few changes originally intended

**Comment by jonboiser** [comment]
@Ali-Kazmi It looks like you might have rebased on `develop` at a past time. I think maybe you needed to fetch the latest stuff from our remote and rebase on `learningequality/kolibri@develop` if that makes sense. If everything is done correctly, your PR should only have commits authored by you.

What I'll cherry pick your commit directly to `develop`. Thanks for the help on this issue!

**Comment by jonboiser** [comment]
Fixed in https://github.com/learningequality/kolibri/commit/d511d12ee907ce0efadc1adb43bfeff2eb7a9529

**Pull request closed without merging**